.github/workflows/e2e_tests.yaml

```diff
@@ -0,0 +1,24 @@
+name: e2e tests
+on:
+  push:
+    branches:
+      - staging
+      # delete after merge
```

Comment:
Maybe you want to put a TODO here in order to not forget it. Or simply delete it already now, since it is working fine.

Reply:
ok, removed it as it's working fine